pkg/apis/management.cattle.io/v3/authn_types.go

```diff
@@ -421,6 +421,7 @@ type LdapFields struct {
 	GroupMemberMappingAttribute     string   `json:"groupMemberMappingAttribute,omitempty"     norman:"default=member,notnullable,required"`
 	ConnectionTimeout               int64    `json:"connectionTimeout,omitempty"               norman:"default=5000,notnullable,required"`
 	NestedGroupMembershipEnabled    bool     `json:"nestedGroupMembershipEnabled"              norman:"default=false"`
+	SearchUsingServiceAccount       bool     `json:"searchUsingServiceAccount,omitempty"       norman:"default=false"`
```

Comment:
I don't think you need `omitempty`.

Boolean `false` is regarded as empty, which would be omitted, but in this case, we would default to false anyway?